README.adoc

```diff
@@ -29,22 +29,17 @@ You will learn how to use MicroProfile (MP) Fault Tolerance to build resilient m
 that reduce the impact from failure and ensure continued operation of services.
 
 MP Fault Tolerance provides a simple and flexible solution to build fault-tolerant microservices.
-Fault tolerance leverages different strategies to guide the execution and result of logic.
-As stated in the https://microprofile.io/project/eclipse/microprofile-fault-tolerance[MicroProfile website],
-retry policies, bulkheads, and circuit breakers are popular concepts in this area.
-They dictate whether and when executions take place, and fallbacks offer an alternative result
-when an execution does not complete successfully.
-
-The application that you will be working with is an `inventory` service, which collects,
-stores, and returns the system properties.
-It uses the `system` service to retrieve the system properties for a particular host.
-You will add fault tolerance to the `inventory` service so that it reacts accordingly when the `system`
-service is unavailable.
-
-You will use the `@Fallback` annotations from the MicroProfile Fault Tolerance
+You will build the fault-tolerance microserivce with the
+`@Fallback` annotation from the MP Fault Tolerance
```

Comment:
I'm not sure if "MP" is familiar enough that users will automatically know what it stands for. While we don't need to spell out abbreviations like HTTP or URL, I'd say "MicroProfile (MP) Fault Tolerance" here. :)

Reply:
The "MicroProfile (MP)" was introduced at the very beginning of the guide, thus it looks ok to use the abbreviation here.